vtm/src/org/oscim/tiling/source/mapfile/MapDatabase.java

```diff
@@ -1113,10 +1143,10 @@ private MapReadResult readMapData(Tile upperLeft, Tile lowerRight, Selector sele
         int[] labelPosition = new int[2];
 
         /* get the label position latitude offset (VBE-S) */
-        labelPosition[1] = mReadBuffer.readSignedInt();
+        labelPosition[0] = mReadBuffer.readSignedInt();
```

Comment:
Please let's stay focused on pull request's purpose and avoid changing other code areas.

Furthermore we need to stay compatible / similar with [Mapsforge code](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-map-reader/src/main/java/org/mapsforge/map/reader/MapFile.java#L935) which is the origin.

Reply:
agree. will remove those changes